lib/service_manager.dart

```diff
@@ -64,23 +65,23 @@ class ServiceConnectionManager {
       _stateController.add(null);
       _connectionAvailableController.add(service);
 
-      _isolateManager._initIsolates(vm.isolates);
+      await _isolateManager._initIsolates(vm.isolates);
       service.onIsolateEvent.listen(_isolateManager._handleIsolateEvent);
       service.onExtensionEvent
           .listen(_serviceExtensionManager._handleExtensionEvent);
 
-      onClosed.then((_) => vmServiceClosed());
-
-      service.streamListen('Stdout');
-      service.streamListen('Stderr');
-      service.streamListen('VM');
-      service.streamListen('Isolate');
-      service.streamListen('Debug');
-      service.streamListen('GC');
-      service.streamListen('Timeline');
-      service.streamListen('Extension');
-      service.streamListen('_Graph');
-      service.streamListen('_Logging');
+      unawaited(onClosed.then((_) => vmServiceClosed()));
+
+      unawaited(service.streamListen('Stdout'));
```

Comment:
should these listen calls really be unawaited? This seems like something that is reasonable to await.
For efficiency you could accumulate a list of all the streamListen future calls and
then wait for all to be done.

Reply:
done.